config/7.5.0/envs/rhdm-authoring-ha.yaml

```diff
@@ -46,165 +46,6 @@ console:
 # ES/AMQ BEGIN
 others:
   - roles:
```

Comment:
any reason this roles array is still here at all? looks like you deleted everything underneath it?

Reply:
If I remove that this issue happens:

`--- FAIL: TestRhdmAuthoringHAEnvironment (0.05s)
    defaults_test.go:212: 
                Error Trace:    defaults_test.go:212
                Error:          Expected nil, but got: &errors.errorString{s:"error unmarshaling JSON: json: cannot unmarshal object into Go struct field Environment.others of type []v2.CustomObject"}
                Test:           TestRhdmAuthoringHAEnvironment
                Messages:       Error getting prod environment
panic: runtime error: index out of range [recovered]
        panic: runtime error: index out of range
`

Reply:
maybe a indentation error?

Reply:
Leave the dash only